docs/dev/annual_updates.rst

```diff
@@ -249,6 +260,10 @@ script in the terminal. From within the pudl repo directory, run:
 
     python src/pudl/transform/ferc1.py
 
+C. EPA CEMS
+^^^^^^^^^^^
+
+**4.C.1)** Use dagster to materialize the ``epacems`` asset group and debug.
```

Comment:
The only thing to mention here is the need to manually add plants without timezones into the additional_epacems_plants.csv. Would be helpful to quickly explain what this CSV is and how to add plants into it:

The most common errors will occur when added CEMS plants lack timezone data in the EIA database. To fix this, we manually add these plants to `pudl.package_data.epacems.additional_epacems_plants.csv`. Add the plant ID from the warning error, and look it up first in the CAMD web query and subsequently online if needed to confirm its location. Add this as a new entry into the database and rematerialize the asset group.